docs/_data/team.js

```diff
@@ -0,0 +1,150 @@
+export const core = [
+  {
+    avatar: 'https://www.github.com/yyx990803.png',
+    name: 'Evan You',
+    title: 'Creator',
+    org: 'Vue.js',
+    orgLink: 'https://vuejs.org/',
+    desc: 'Husband, father of two, independent open source developer.',
+    links: [
+      { icon: 'github', link: 'https://github.com/yyx990803' },
+      { icon: 'twitter', link: 'https://twitter.com/youyuxi' }
+    ],
+    sponsor: 'https://github.com/sponsors/yyx990803'
+  },
+  {
+    avatar: 'https://www.github.com/patak-dev.png',
+    name: 'Patak',
+    title: 'A collaborative being',
+    org: 'StackBlitz',
+    orgLink: 'https://stackblitz.com/',
+    desc: 'Core team member of Vite. Team member of Vue.',
+    links: [
+      { icon: 'github', link: 'https://github.com/patak-dev' },
+      { icon: 'twitter', link: 'https://twitter.com/patak_dev' }
+    ],
+    sponsor: 'https://github.com/sponsors/patak-dev'
+  },
+  {
+    avatar: 'https://www.github.com/antfu.png',
+    name: 'Anthony Fu',
+    title: 'A fanatical open sourceror',
+    org: 'NuxtLabs',
+    orgLink: 'https://nuxtlabs.com/',
+    desc: 'Core team member of Vite & Vue Working at NuxtLabs.',
+    links: [
+      { icon: 'github', link: 'https://github.com/antfu' },
+      { icon: 'twitter', link: 'https://twitter.com/antfu7' }
+    ],
+    sponsor: 'https://github.com/sponsors/antfu'
+  },
+  {
+    avatar: 'https://github.com/sodatea.png',
+    name: 'Haoqun Jiang',
+    title: 'Developer',
+    org: 'Vue.js',
+    orgLink: 'https://vuejs.org/',
+    desc: 'Vite/Vite core team member. Full-time open sourcerer.',
+    links: [
+      { icon: 'github', link: 'https://github.com/sodatea' },
+      { icon: 'twitter', link: 'https://twitter.com/haoqunjiang' }
+    ],
+    sponsor: 'https://github.com/sponsors/sodatea'
+  },
+  {
+    avatar: 'https://github.com/Shinigami92.png',
+    name: 'Shinigami',
+    title: 'Maintainer',
+    org: 'Faker',
+    orgLink: 'https://fakerjs.dev',
+    desc: 'Passionate TypeScript enthusiast working extensively with Vue SPA and pug.',
+    links: [
+      { icon: 'github', link: 'https://github.com/Shinigami92' },
+      { icon: 'twitter', link: 'https://twitter.com/Shini_92' }
+    ],
+    sponsor: 'https://github.com/sponsors/Shinigami92'
+  },
+  {
+    avatar: 'https://github.com/aleclarson.png',
+    name: 'aleclarson',
```

Comment:
@aleclarson just confirming if you'd be presenting your name as `aleclarson` instead of `Alec Larson`?

Reply:
change it please ❤️

Reply:
and use this picture
https://drive.google.com/file/d/1ex64BOY7yYQ9oAmWLzBELPlvzqIKjlhn/view?usp=sharing

Reply:
```suggestion
    name: 'Alec Larson',
```